frontends/web/src/components/anchor/anchor.tsx

```diff
@@ -39,25 +41,48 @@ type TProps = {
  *
  * @param {LinkProps} props - The props object containing properties for the Link component.
  */
+
+type TUseConfirm = [
+    (title: string, message: string) => Promise<boolean>,
+    React.FunctionComponent
+];
+
 export const A = ({
   href,
   icon,
   className,
   children,
   ...props
 }: TProps) => {
+  const { t } = useTranslation();
+  const [ getConfirmation, Confirmation ]: TUseConfirm = useConfirm();
+
+  const onReject = async (reason: string) => {
+    if (reason.indexOf('Blocked') === -1) {
+      return;
+    }
+
+    const status = await getConfirmation(t('blockedOpen.title'), t('blockedOpen.message'));
+    if (status) {
+      window.open(href, '_blank');
```

Comment:
This does and should not work in Qt/WebView. This is what `import { open } from 'api/system.ts';` is for,

Reply:
ah yes, I though so but still wanted to give it a shoot :) However I could not test this because of the issue with the dialog not even opening/showing.

Is there the possibility to open a link without `api/system.tx open` ? If not then my whole approach does not really work anyway...I wanted to use `open` and then if it [fails](https://github.com/digitalbitbox/bitbox-wallet-app/pull/2333/files#diff-a0758e6db02536ac8e041b50367493f59198a892c667b35d768d2cfc7da9b497R61) because it was blocked the user should confirm opening it anyway in the dialog.

Reply:
(just checked it looks like we need the backend to open any url)